templates/signup.tpl

```diff
@@ -13,74 +13,56 @@
+        <div class="panel-body">
             <form method="post">
-                <div class="form-group">
-                    <label class="lb text-warning" >Email address</label>
-                    <input type="email" class="form-control form-control-lg" id="email" name="email" aria-describedby="emailHelp" placeholder="Enter Email">
+                <div class="form-group form-auth">
+                    <label class="lb text-warning">Email address</label>
+                    <input type="email" class="form-control form-control-lg" id="email" name="email"
+                           aria-describedby="emailHelp" placeholder="Enter Email">
                 </div>
                 <div class="form-group">
-                    <label class="lb text-warning" >Password</label>
-                    <input type="password" class="form-control form-control-lg" id="password" name="password" placeholder="Enter Password">
+                    <label class="lb text-warning">Password</label>
+                    <input type="password" class="form-control form-control-lg" id="password" name="password"
+                           placeholder="Enter Password">
                 </div>
                 <div class="form-group">
-                    <label class="lb text-warning" >Confirm Password</label>
-                    <input type="password" class="form-control form-control-lg" id="confirmPassword" name="confirmPassword" placeholder="Enter Password">
+                    <label class="lb text-warning">Confirm Password</label>
+                    <input type="password" class="form-control form-control-lg" id="confirmPassword"
+                           name="confirmPassword" placeholder="Enter Password">
                 </div>
                 <div class="form-group">
-                    <label class="lb text-warning" >Name</label>
-                    <input type="text" class="form-control form-control-lg" id="name" name="name" placeholder="Enter Name">
+                    <label class="lb text-warning">Name</label>
+                    <input type="text" class="form-control form-control-lg" id="name" name="name"
+                           placeholder="Enter Name">
                 </div>
                 <div class="form-group">
-                    <label class="lb text-warning" >Phone</label>
-                    <input type="number" class="form-control form-control-lg" id="phone" name="phone" placeholder="Enter Phone">
+                    <label class="lb text-warning">Phone</label>
+                    <input type="number" class="form-control form-control-lg" id="phone" name="phone"
+                           placeholder="Enter Phone">
                 </div>
                 <div class="text-danger font-weight-bold">{if isset($error) }{$error}{/if}</div>
-                <button class="btn btn-outline-warning btn-">Sign Up</button><br>
+                <button class="btn btn-outline-warning btn-">Sign Up</button>
```

Comment:
You should define the wording instead of input directly in the source code (Sign Up)